packages/react-native-win32/src/Libraries/Components/Text/TextWin32.Props.ts

```diff
@@ -1,15 +1,19 @@
 import RN = require('react-native');
-import { SharedAccessibilityPropsIOSandWin32, OmittedAccessibilityPropsWin32, BasePropsWin32 } from '../View/ViewWin32.Props';
+import { SharedAccessibilityPropsIOSandWin32, OmittedAccessibilityPropsWin32, BasePropsWin32, IKeyboardEvent, IHandledKeyboardEvent } from '../View/ViewWin32.Props';
 
 // removes from interface T the members of interface K
 type Omit<T, K> = Pick<T, Exclude<keyof T, keyof K>>;
 
-export type TextWin32OmitTypes = RN.TextPropsAndroid &
+export type TextWin32OmitTypes = Omit<RN.TextPropsAndroid, SharedTextPropsAndroidandWin32> &
   RN.TextPropsIOS &
   RN.AccessibilityPropsAndroid &
   Omit<RN.AccessibilityPropsIOS, SharedAccessibilityPropsIOSandWin32> &
   OmittedAccessibilityPropsWin32;
 
+export type SharedTextPropsAndroidandWin32 = {
```

Comment:
Just checking if intentional that it is `and` and not `And`.

Reply:
I had written it this way to match `SharedAccessibilityPropsIOSandWin32` but it looks off so I updated it to `And`

Reply:
I actually decided to change it back so it remains consistent with what's in master and 0.64